CLAUDE.md

```
@@ -829,59 +829,509 @@ Response
+  AND a.id <> ALL(:deck_asset_ids)
+  AND a.id <> ALL(:shown_asset_ids)
+  AND a.id >= :rand_id
+ORDER BY a.id
+LIMIT 20;
+```
+
+#### Random Sampling Strategy (PK Range Random)
+
+**ORDER BY random() 금지**: 대용량 테이블에서 성능 최악
+
+**MVP 추천: PK 범위 랜덤**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)